src/quo/animated.cljs

```diff
@@ -16,6 +17,7 @@
 (def scroll-view (reagent/adapt-react-class (.-ScrollView animated)))
 (def code (reagent/adapt-react-class (.-Code animated)))
 (def animated-flat-list (create-animated-component gh/flat-list-raw))
+(def blur-view (create-animated-component (.-BlurView blur)))
```

Comment:
Just a question - the `blur-view` component is already available under the namespaces
1. `status-im.ui.components.react`
2. `react-native.core`

now, we are adding one more in `quo.animated`. 

Based on the [new guidelines](https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#project-structure), I believe this should live under the `react-native` directory.

Which would be the correct namespace or structure/categorization for the React Native dependencies (BlurView, MaskedView, FastImage,...etc)? 



@flexsurfer @ilmotta

Reply:
That's an animated blur view.

Reply:
I think so @smohamedjavid. This PR should use only `react-native.reanimated` and the `react-native.core/blur-view` as you said. We shouldn't rely on `quo.animated` anymore for the new app code.